makefile.shade

```diff
@@ -20,7 +20,9 @@ functions
         const string UniverseCommitsFileName = "commits-universe";
 
         private static bool Quiet { get; set; }
-
+        private static bool UseGateBranch { get; set; }
+        private const string GateBranch = "gateDev";
```

Comment:
Also, if you make it an environment variable, you don't need the new build target. If the variable is set sync to it, otherwise do nothing